client/my-sites/drafts/draft-list.jsx

```diff
@@ -1,62 +1,93 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	PureRenderMixin = require( 'react-pure-render/mixin' ),
-	omit = require( 'lodash/omit' );
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import omit from 'lodash/omit';
 
 /**
  * Internal dependencies
  */
-var PostListFetcher = require( 'components/post-list-fetcher' ),
-	hasTouch = require( 'lib/touch-detect' ).hasTouch,
-	actions = require( 'lib/posts/actions' ),
-	Draft = require( 'my-sites/draft' ),
-	EmptyContent = require( 'components/empty-content' ),
-	infiniteScroll = require( 'lib/mixins/infinite-scroll' ),
-	observe = require( 'lib/mixins/data-observe' );
-
-var DraftList = React.createClass( {
-
-	mixins: [ PureRenderMixin ],
+import AuthorSelector from './author-selector';
+import Count from 'components/count';
+import Draft from 'my-sites/draft';
+import EmptyContent from 'components/empty-content';
+import PostListFetcher from 'components/post-list-fetcher';
+import QueryPostCounts from 'components/data/query-post-counts';
+import {
+	getNormalizedPostCounts,
+	getNormalizedMyPostCounts,
+	isRequestingPostCounts
+} from 'state/posts/counts/selectors';
+import actions from 'lib/posts/actions';
+import { hasTouch } from 'lib/touch-detect';
+import infiniteScroll from 'lib/mixins/infinite-scroll';
+import observe from 'lib/mixins/data-observe';
+import userLib from 'lib/user';
```

Comment:
`lib/user` is deprecated. For your usage in this component, you should look to [the `getCurrentUserId`](https://github.com/Automattic/wp-calypso/blob/9853eb5f87cfe331140943385ba7b989a2833429/client/state/current-user/selectors.js#L8-L16) selector.